pytradfri/api/aiocoap_api.py

```diff
@@ -203,6 +248,14 @@ def error_callback(ex):
 
         return self._psk
 
+    @classmethod
```

Comment:
Given that an APIFactory will be created for each gateway, rather than each observation, should this be instanced instead?

Reply:
Since this function is called from the callback defined in `_observe()`, when trying to use an instance method, the callback doesn't have any relations to `self` or an instance of `APIFactory` anymore. That's why I put this on a 'global' timer.

If you can point me out on how to use an instance method in the aforementioned callback, I would gladly implement that change.

Reply:
`success_callback`  only has it's own scope within the method.